docs/guides/migrating-from-camunda-platform-7/adjusting-bpmn-models.md

```diff
@@ -0,0 +1,199 @@
+---
+id: adjusting-bpmn-models
+title: Adjusting BPMN models
+description: "Learn how to adjust your BPMN models when migrating from Camunda Platform 7 to Camunda Platform 8."
+---
+
+Ensure your BPMN process models are adjusted as follows to migrate them from Camunda Platform 7 to Camunda Platform 8:
+
+- The namespace of extensions has changed from `http://camunda.org/schema/1.0/bpmn` to `http://camunda.org/schema/zeebe/1.0`.
+- Different configuration attributes are used
```

Comment:
This seems a bit vague -- could we clarify the ambiguity here?

Reply:
They are outlined below based on the BPMN element. Do you have a recommendation for alluding to the coming info on configuring attributes below?

Reply:
I have a recommendation in the commit I'm working on, but it's also a bit odd.

> - Different configuration attributes are used between the platform versions.

Reply:
How about "Different configuration attributes are used between platform versions, as described for each BPMN element below."?